app/controllers/users_controller.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
```

Comment:
Instead of tambahin `authorize!` di satu file, coba look into `load_and_authorize_resource`